apps/nextjs-website/src/editorialComponents/Newsroom/Newsroom.tsx

```diff
@@ -142,9 +143,8 @@ const Newsroom = (props: INewsroom) => {
           flexWrap: { xs: 'nowrap', md: 'wrap' },
           maxWidth: { md: '1280px', lg: '1310px' },
           mx: 'auto',
-          marginLeft: { sm: '-32px', md: '-16px', lg: 'auto' },
-          'div.MuiGrid-item:first-of-type': {
-            marginLeft: { xs: '32px', sm: '16px', md: '0px' },
+          '&.MuiGrid-container': {
+            marginLeft: { md: '-24px' },
```

Comment:
The three cards do not occupy all the available space. If you look at the [design on Figma](https://www.figma.com/file/90VOD5Vgg8k0h6QAIC5FvE/23-Q2---DevPortal?type=design&node-id=3231-60563&mode=design&t=5r0bCLOAAGjzn9eY-4), the cards should be aligned to the title on the left side (as you did) and should fit all the width of the column:
![image](https://github.com/pagopa/developer-portal/assets/39835990/b5e19b11-ef11-488d-a2a6-a116b67c4884)

![image](https://github.com/pagopa/developer-portal/assets/39835990/d56fbffd-28fe-4251-a00d-a69657f492c4)